### PR TITLE
feat(pci): add guide advice for windows private mode vrack

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11052
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~ [n/a}

## Description

In public cloud, when a user create a windows instance in private mode, display a banner message with a link to an OVH guide.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js [hunk 1]
```diff
@@ -17,6 +17,32 @@ export const INSTANCE_READ_MORE_GUIDE = {
...
+    'https://docs.ovh.com/es/public-cloud/activate-windows-licence-private-mode-instance/',
+  it_IT:
+    'https://docs.ovh.com/it/public-cloud/activate-windows-licence-private-mode-instance/',
+  pt_PT:
+    'https://docs.ovh.com/pt/public-cloud/activate-windows-licence-private-mode-instance/',
+  ASIA:
+    'https://docs.ovh.com/asia/en/public-cloud/activate-windows-licence-private-mode-instance/',
+  DEFAULT:
+    'https://docs.ovh.com/gb/en/public-cloud/activate-windows-licence-private-mode-instance/',
+};
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js:45]: **nitpick:** Maintain the banner based on the User Sub and not the user selected language.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js [hunk 2]
```diff
@@ -56,6 +57,10 @@ export default class PciInstancesAddController {
     this.instanceReadMoreUrl =
       INSTANCE_READ_MORE_GUIDE.ALL_GUIDE[this.user.ovhSubsidiary] ||
       INSTANCE_READ_MORE_GUIDE.ALL_GUIDE.DEFAULT;
+    this.windowsPrivateModeLicenseGuideUrl =
+      WINDOWS_PRIVATE_MODE_LICENSE_GUIDE[this.user.language] ||
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js:61]: Usually we only use the subsidiary and not the language. You also need to modify the constants file.

**Comment by ghyenne** [comment]
You can clean up the branch (commit / revert commit is not necessary on BAU topics). I think you need only one commit.

File: packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js (hunk 1 above)
Comment (qpavy) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js:45]: How do you manage url like fr_CA, en_CA, en_GB, en_US, en_AU if you use sub ?

File: packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js (hunk 2 above)
Comment (qpavy) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js:61]: How do you manage url like fr_CA, en_CA, en_GB, en_US, en_AU if you use sub ?

File: packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js (hunk 1 above)
Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js:45]: QC:
CA:
GB:
US:
AU:

File: packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js:61]: In constant file: 
https://github.com/ovh/manager/pull/9421#discussion_r1223925011

File: packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js (hunk 1 above)
Comment (qpavy) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js]: Done

File: packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js (hunk 2 above)
Comment (qpavy) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js]: Done

File: packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js [hunk 3]
```diff
@@ -17,6 +17,32 @@ export const INSTANCE_READ_MORE_GUIDE = {
       'https://docs.ovh.com/gb/en/public-cloud/attaching-pci-floating-ip-to-instance/',
   },
 };
+export const WINDOWS_PRIVATE_MODE_LICENSE_GUIDE = {
+  EN:
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js:21]:
```suggestion
  GB:
```

**Review by ghyenne — CHANGES_REQUESTED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**